devices/oneplus-bacon/kernel/default.nix

```diff
@@ -0,0 +1,62 @@
+{
+  mobile-nixos
+, fetchFromGitHub
+, kernelPatches ? [] # FIXME
+, buildPackages
+, dtbTool
+}:
+
+(mobile-nixos.kernel-builder-gcc6 {
+  configfile = ./config.armv7l;
+
+  file = "zImage-dtb";
+  hasDTB = false;
+
+  version = "3.4.113";
+  src = fetchFromGitHub {
+    owner = "LineageOS";
+    repo = "android_kernel_oppo_msm8974";
+    rev = "a160ed58b3b94f63d1983b1798e280ed6602656a"; # lineageos-16.0
+    sha256 = "sha256-odpo3vnQ1M2eWzqEFrXtipQZxlL5ko0/tfDM5yUplMc=";
+  };
+  patches = [
+    #./99_framebuffer.patch
```

Comment:
Since it's at the end of this list you can remove this commented-out instance :).